src/components/list/README.md

```diff
@@ -0,0 +1,126 @@
+# md-list
+
+`md-list` is a container component that wraps and formats a series of line items. As the base list component,
+ it provides Material Design styling, but no behavior of its own.
+
+## Usage
+
+### Simple list
+
+To use `md-list`, first import the list directives and add them to your component's directives array:
+
+```javascript
+@Component({
+  ...
+  directives: [MD_LIST_DIRECTIVES]
+})
+```
+
+In your template, create an `md-list` element and wrap each of your items in an `md-list-item` tag.
+
+```html
+<md-list>
+   <md-list-item> Pepper </md-list-item>
+   <md-list-item> Salt </md-list-item>
+   <md-list-item> Paprika </md-list-item>
+</md-list>
+```
+
+Output:
+
+<img src="https://material.angularjs.org/material2_assets/list/basic-list.png">
+
+### Multi-line lists
+
+If your list requires multiple lines per list item, annotate each line with an `md-line` attribute. 
+You can use whichever heading tag is appropriate for your DOM hierarchy (doesn't have to be `h3`), 
+as long as the `md-line` attribute is included.
+
+```html
+<!-- two line list -->
+<md-list>
+  <md-list-item *ngFor="#message of messages">
+    <h3 md-line> {{message.from}} </h3>
+    <p md-line>
+      <span> {{message.subject}} </span>
+      <span class="demo-2"> -- {{message.message}} </span>
+    </p>
+  </md-list-item>
+</md-list>
+
+<!-- three line list -->
+<md-list>
+  <md-list-item *ngFor="#message of messages">
+    <h3 md-line> {{message.from}} </h3>
+    <p md-line> {{message.subject}} </p>
+    <p md-line class="demo-2"> {{message.message}} </p>
+  </md-list-item>
+</md-list>
+```
+
+Two line list output:
+
+<img src="https://material.angularjs.org/material2_assets/list/two-line-list.png">
+
+Three line list output: 
+
+<img src="https://material.angularjs.org/material2_assets/list/three-line-list.png">
+
+### Lists with avatars
+
+To include an avatar, add an image tag with an `md-list-avatar` attribute. 
+
+```html
+<md-list>
+  <md-list-item *ngFor="#message of messages">
+    <img md-list-avatar src="..." alt="...">
+    <h3 md-line> {{message.from}} </h3>
+    <p md-line>
+      <span> {{message.subject}} </span>
+      <span class="demo-2"> -- {{message.message}} </span>
+    </p>
+  </md-list-item>
+</md-list>
+```
+
+Output:
+
+<img src="https://material.angularjs.org/material2_assets/list/list-with-avatar.png">
```

Comment:
Go ahead and use some of the angular team avatars, rather than that card (can steal them from [here](https://angular.io/about/)).

Also, aren't avatars supposed to be a circle?